src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,140 @@
+package duke;
+
+import duke.exceptions.DukeInvalidArgumentException;
+import duke.exceptions.DukeMissingArgumentException;
+import duke.task.Tasklist;
+import java.util.Scanner;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
```

Comment:
I like that you have provided a lot of documentations for fields, methods and classes.